docs/developer-guide/syntax-reference.rst

```diff
@@ -446,12 +446,18 @@ The WITH clause for the result supports the following properties:
 |                         | set, then the format of the input stream/table is used.                                              |
 +-------------------------+------------------------------------------------------------------------------------------------------+
 | PARTITIONS              | The number of partitions in the backing topic. If this property is not set, then the number          |
-|                         | of partitions is taken from the value of the ``ksql.sink.partitions`` property, which                |
-|                         | defaults to four partitions. The ``ksql.sink.partitions`` property can be set in the                 |
-|                         | properties file the KSQL server is started with, or by using the ``SET`` statement.                  |
+|                         | of partitions of the input stream/table will be used. In the Join queries the properties would       |
```

Comment:
"In join queries, the property values are taken from the left-side stream or table."

I'd make the edits directly, but that would mess up the table alignment. :)